bower.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "json-refs",
   "main": "browser/json-refs.js",
-  "version": "1.3.0",
```

Comment:
I'm not sure about this change.  My understanding is that Bower uses tags just like NPM does so keeping that as-is shouldn't be a problem.  I don't usually rev `bower.json` or `package.json` in `master` until it's time to release but maybe I'm doing it wrong.